views/personal/residence.pug

```diff
@@ -0,0 +1,29 @@
+extends ../base
+include ../_includes/yesNoRadios
+
+block variables
+  -var title = 'Confirm if you are an Ontario resident'
```

Comment:
You can no longer be an Ontario resident and still use this (as long as you lived here last year). But this is more clear for the purposes of our demo so it's cool.